x-pack/plugins/aiops/public/components/log_categorization/category_table/table_header.tsx

```diff
@@ -49,32 +47,57 @@ export const TableHeader: FC<Props> = ({
           </EuiText>
         </EuiFlexItem>
         {selectedCategoriesCount > 0 ? (
-          <>
-            <EuiFlexItem grow={false}>
-              <EuiButtonEmpty
-                data-test-subj="aiopsLogPatternAnalysisOpenInDiscoverIncludeButton"
-                size="s"
-                onClick={() => openInDiscover(QUERY_MODE.INCLUDE)}
-                iconType="plusInCircle"
-                iconSide="left"
-              >
-                {labels.multiSelect.in}
-              </EuiButtonEmpty>
-            </EuiFlexItem>
-            <EuiFlexItem grow={false}>
-              <EuiButtonEmpty
-                data-test-subj="aiopsLogPatternAnalysisOpenInDiscoverExcludeButton"
-                size="s"
-                onClick={() => openInDiscover(QUERY_MODE.EXCLUDE)}
-                iconType="minusInCircle"
-                iconSide="left"
-              >
-                {labels.multiSelect.out}
-              </EuiButtonEmpty>
-            </EuiFlexItem>
-          </>
+          <EuiFlexItem grow={false}>
+            <OpenInDiscoverButtons openInDiscover={openInDiscover} />
+          </EuiFlexItem>
         ) : null}
       </EuiFlexGroup>
     </>
   );
 };
+
+export const OpenInDiscoverButtons: FC<{ openInDiscover: OpenInDiscover; showText?: boolean }> = ({
+  openInDiscover,
+  showText = true,
+}) => {
+  const { labels, openFunction } = openInDiscover;
+  const TooltipWrapper: FC<PropsWithChildren<{ text: string }>> = ({ text, children }) => {
```

Comment:
Should we move `TooltipWrapper` component out and pass showText as a prop so this is not created every time OpenInDiscoverButtons is rendered?

Reply:
Updated in https://github.com/elastic/kibana/pull/178916/commits/a6b73baba7bab7bb71a1e267c652b1e4253c5ffe